data/human/news.txt

```diff
@@ -365,6 +373,7 @@ news "groundskeeper"
 			"They told me that they wanted the landscaping to be classic and entertaining, so I constructed a giant hedge maze."
 			"Despite all our best terraforming efforts, it can still be hard to get native Earth plants to grow on an alien world."
 			"My employer once asked me to add a garden in their Star Queen. The pirates were sure confused when they boarded us."
+			"My employer owns almost 2 complete planets, but today I'm watering the plans in the small appartment she owns on this planet."
```

Comment:
Did you mean to say she is watering the plans here, or should that actually be "plants"? Also mis-spelling of apartment.

```suggestion
			"My employer owns almost 2 complete planets, but today I'm watering the plants in the small apartment she owns on this planet."
```